### ISSUE TITLE
Dgraph live import skips long text tuples

### ISSUE THREAD

**Issue by advancedlogic** [dgraph-io/dgraph#2163]
I am using v.1.0.3 and I am moving data from v.0.8. I had to change something in the schema and in the tuples (got rid of ^^<pwd:password>). 
My schema has 
```
document.title:          string @index(term, fulltext, trigram) .
document.content:        string @index(fulltext,trigram) .
```

I run dgraph live -r tuples.rdf -s schema.rdf and everything run smoothly but in the running dgraph instance I have:

_2018/02/25 14:42:30 node.go:400: WARN: A tick missed to fire. Node blocks too long!_

When I run a query to retrieve the collection of documents, part of the results have no document.content even if they are present in the file I have imported.

This is critical for me at the moment. Any workaround?

**Comment by advancedlogic** [comment]
Yes it completed loading. Unfortunately I cannot share the logs but as I can see dgraph live skips long text. Test was done on an SSD drive on OS X.

**Comment by pawanrawal** [comment]
Could you share a reproducible example? I tried with a long line with > 25000 chars and it was loaded just fine.

**Comment by pawanrawal** [comment]
This might be related to https://github.com/dgraph-io/dgraph/pull/2188. I am going to merge the PR today. It would be great if you can try after that.

**Comment by pawanrawal** [comment]
I have merged #2188 and am going to close this issue for now. Please try with the master and if it is still an issue, feel free to reopen.
